2312/2/battle.rb

```diff
@@ -0,0 +1,25 @@
+require './word'
+require './lyrics'
+
+# this class analyzes text of battle
+class Battle
+  def initialize(battle_file_path)
+    @battle_file_path = battle_file_path
+  end
+
+  def extract_battlers_names
```

Comment:
Again, think about business entities when writing OOP code.
Medidate on this code.

`battle#battlers_names` vs `battle#extract_battles_names`
`battle#rounds_count` vs `battle#count_rounds`